### PR TITLE
data: jegrami

### PR DESCRIPTION
**Pull request by jegrami**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7399"></a>

### CODE REVIEW

File: data/jegrami.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+ {
```
Comment (kumarsonsoff3) [inline data/jegrami.json:1]: You're missing some required properties link `name`, `bio`, etc. Please read the [QuickStart Docs](https://linkfree.io/docs/quickstart) for more information on how to add Your Profile. 

Please add your name, and bio as suggested below.

```suggestion
 {
  "name": "Jeremiah Igrami",
  "bio": "YOUR__BIO",
  "links": [
    {
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by jegrami — COMMENTED** [review]
Thanks for the change! What should I do next?

File: data/jegrami.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "name": "Jeremiah Igrami",
+  "bio": "YOUR__BIO",
```
Comment (kumarsonsoff3) [inline data/jegrami.json:3]: Please add your bio/about/headline here instead in place of `YOUR__BIO` placeholder.

**Review by jegrami — COMMENTED** [review]
I have made the correction. Thanks again.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for Contributing! 🎉 

<p align="center"><a href="https://linkfree.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**